tests/conftest.py

```diff
@@ -98,6 +98,8 @@ def db_session():
     logging.debug("Database path %s", os.environ['FLASK_SQLALCHEMY_DATABASE_URI'])
     _app = create_app()
     with _app.app_context():
+        db.drop_all()
+        db.create_all()
```

Comment:
Can you explain this change?

Reply:
I wanted to be sure that the DB was completely empty when this fixture started working.

Reply:
This was just for testing